<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileConvertForm</class>
 <widget class="QWidget" name="FileConvertForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_ProtocolSelect"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_FileSelect">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_FileConvert">
          <property name="text">
           <string>转换文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Clear">
          <property name="text">
           <string>清空窗口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CustomComboBox" name="comboBox_HistoryFile"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>customcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
